ipam/fileIpam.go

```diff
@@ -50,36 +52,45 @@ type IPAddress struct {
 	IsPrimary bool
 }
 
-// Creates the MAS source.
-func newMasSource(options map[string]interface{}) (*masSource, error) {
+// Creates the MAS/fileIPAM source.
+func newFileIpamSource(options map[string]interface{}) (*fileIpamSource, error) {
 	var filePath string
 	if runtime.GOOS == windows {
 		filePath = defaultWindowsFilePath
 	} else {
 		filePath = defaultLinuxFilePath
 	}
 
-	return &masSource{
-		name: name,
-		filePath: filePath,
-	}, nil
+	environment, _ := options[common.OptEnvironment].(string)
+
+	if environment == common.OptEnvironmentMAS {
+		return &fileIpamSource{
```

Comment:
Assign variable name using if-else and create fileIpamSource once.

Reply:
updated.